pkg/kritis/review/review.go

```diff
@@ -158,7 +140,7 @@ func (r Reviewer) handleViolations(image string, pod *v1.Pod, violations []secur
 			errMsg = fmt.Sprintf("%s is not a fully qualified image", image)
 		}
 	}
-	if err := r.vs.HandleViolation(image, pod, violations); err != nil {
+	if err := r.config.Strategy.HandleViolation(image, pod, violations); err != nil {
```

Comment:
We can fix this in a subsequent change, but this should be called HandleViolations, as it accepts more than one violation.